docs/src/main/sphinx/functions/setdigest.rst

```diff
@@ -0,0 +1,154 @@
+=========================
+Set Digest functions
+=========================
+
+Trino offers several functions that deal with
+`MinHash <https://en.wikipedia.org/wiki/MinHash>`_ technique.
+
+MinHash is a technique  which is employed to quickly estimate the
+`Jaccard similarity coefficient <https://en.wikipedia.org/wiki/Jaccard_index>`_
+between two sets.
+
+One of the common applications of the MinHash technique is in data mining where it is employed for
+detecting at scale near-duplicate web pages. By using this information, the search engines
+efficiently avoid showing two pages which are nearly identical within the search results.
+
+Data structures
+---------------
+
+Trino implements Set Digest data sketches by encapsulating the following components:
+
+- `HyperLogLog <https://en.wikipedia.org/wiki/HyperLogLog>`_
+- `MinHash with a single hash function <http://en.wikipedia.org/wiki/MinHash#Variant_with_a_single_hash_function>`_
+
+The Hyperloglog structure is used for the aproximation of the distinct elements
+in the original set.
+
+The MinHash structure is used to store a low memory footprint signature of the original set.
+The similarity of any two sets is estimated by comparing their signatures.
+
+The Trino type for this data structure is called ``setdigest``.
+
+Trino offers the ability to merge multiple Set Digest data sketches.
+
+The following example showcases how the Set Digest functions are
+employed to estimate the similarity between texts::
+
+    WITH text_input(id, text) AS (VALUES
```

Comment:
This query depends on the changes from the PR https://github.com/trinodb/trino/pull/8295

Once the PR is accepted, also the documentation of `make_set_digest` function needs to be correspondingly adjusted.

Reply:
@findinpath Just wanted to confirm if the changes to documentation you are talking about are done already?